packages/vizbuilder/src/helpers/random.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Returns a random, non-cryptographically secure, unique id string.
+ * @returns {string}
+ */
+export function makeRandomId() {
+  return `id${Math.random()
+    .toString(36)
+    .substr(2, 5)}`;
+}
+
+/**
+ * Chooses one random element from an array.
+ * @template T
+ * @param {Array<T>} list Array with the elements from where to pick one
+ * @returns {T}
+ */
+export function pickOne(list) {
+  return list[Math.floor(Math.random() * list.length)];
+}
+
+/**
+ * Slices a string to
+ * @param {string} string The string to slice
+ * @returns {string}
+ */
+export function sortSlice(string) {
```

Comment:
This function is actually very important.
The virtualized list in the selector doesn't actually sort the elements it's showing, the elements are sorted just once, just before they're saved in the vizbuilder's state. The virtualized list only makes the hierarchy iterating over the title properties, but if they were unsorted, we would have repeated titles.
This function makes a kind of index key for the sort function, and it needs the slicing because of ASCII.
I ended up removing the other two functions, so I'll move this one next to the function that uses it, and delete random.js.

Reply:
done in dff1dcb395fac24455ec5272afcaa2bc3e2bea20